test/e2e/storage-endpoint.integration.test.js

```diff
@@ -0,0 +1,57 @@
+const supertest = require("supertest");
+const { put } = require("../../src/storage/s3");
+const config = require("../../src/storage/config");
+const ropstenDocument = require("../fixtures/certificate.json");
+
+const API_ENDPOINT = "https://api-ropsten.opencerts.io";
+const request = supertest(API_ENDPOINT);
+
+describe("storage endpoint test", () => {
+  it("should throw error forbidden when directly access the document", async () => {
+    const document = { foo: "bar" };
+    const params = {
+      Bucket: config.bucketName,
+      Key: "abc",
+      Body: JSON.stringify({ document })
+    };
+    const uploaded = await put(params);
+
+    await request
+      .get(`${uploaded.Location}`)
+      .set("Content-Type", "application/json")
+      .expect("Content-Type", /json/)
+      .expect(403);
+  }, 5000);
```

Comment:
think you might need to clean up after tests because the s3 bucket doesn't get torn down

Reply:
added remove.